### ISSUE TITLE
Npcs keep dieing...

### ISSUE THREAD

**Issue by Loserman778** [CitizensDev/Citizens#767]
Hey guys, is there a way to disable my npc's from dieing? I couldn't find anything about it. Thanks in advanced
- loserman778

**Comment by JaredTS486** [comment]
They should not die to mobs... Sounds like people are killing them. Just set the re spawn time to a short time like 30 seconds. In my server mobs do not target them.

**Comment by Loserman778** [comment]
Players kill them. The thing is, is that they get their items from killing them, and Heroes EXP and money. So its kinda a problem.

**Comment by JaredTS486** [comment]
Yes so there should be a re spawn timer you can set for the guards.

On Tue, Feb 7, 2012 at 1:28 PM, Loserman778 <
reply@reply.github.com
